release-notes/release-notes58.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: TinyMCE 5.8
+title_nav: TinyMCE 5.8
+description: Release notes for TinyMCE 5.8
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+{{site.productname}} 5.8 was released for {{site.enterpriseversion}} and {{site.cloudname}} on <<`WEEKDAY`, `MMM` `D`<sup>`st|nd|th`</sup>, `YYYY`>>. It includes {{site.productname}} 5.8 and additional changes to premium plugins. These release notes provide an overview of the changes for {{site.productname}} 5.8, including:
+
+- [New features](#newfeatures)
+- [Enhancements](#enhancements)
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [Accompanying Premium Skins and Icon Packs changes](#accompanyingpremiumskinsandiconpackschanges)
+- [General bug fixes](#generalbugfixes)
+- [Security fixes](#securityfixes)
+- [Deprecated features](#deprecatedfeatures)
+- [Known issues](#knownissues)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## New features
+
+The following new features were added for the {{site.productname}} 5.8 release.
+
+### Improved keyboard navigation for the inline boundary feature
```

Comment:
This was placed in the wrong place and we missed it in the review. The should be in the enhancements section instead.

Reply:
Fixed here: [`c25796c` (#1966)](https://github.com/tinymce/tinymce-docs/pull/1966/commits/c25796c80db5ed5998f66c8d759a510435df5475)